### ISSUE TITLE
[Enhancement Request] Posts Widget: Exclude by Category

### ISSUE THREAD

**Issue by slomeli79** [pojome/elementor#2205]
Self-explanatory. Being able to exclude by current post and by post ID is great, but the Posts Widget would be far more versatile (and less frequently passed over in favor of alternatives) for the job if it allowed excluding a category or multiple categories.

**Comment by mbh-fruni** [comment]
Yeah, that would be REALLY great!

**Comment by bainternet** [comment]
@slomeli79 

This should be easy to implement with a query filter which is available in Elementor pro 2.1 (currently in beta #4923 )

give your query an ID (example: `slomeli`) and use the generated filter hook to inject your exclude rules:

```PHP

add_action( 'elementor_pro/posts/query/slomeli', function ( $wp_query, $posts_widget ) {
	$wp_query->set( 'category__not_in', [ 10, 25 ] ); //category id's to exclude posts from
}, 10 ,2 );
```

I merged this issue with  #1748, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
